src/amo/components/SearchForm/index.js

```diff
@@ -125,7 +125,10 @@ export class SearchFormBase extends React.Component {
   }
 
   handleSearchChange = (event) => {
-    this.setState({ searchValue: event.target.value });
+    const searchValue = event.target.value || '';
```

Comment:
Can you add a test case to cover this condition? I think you want to test the case where `event.target.value` is not set or something, and ensure it fallbacks to empty string, avoiding any further issue.